libs/solid/skeleton/src/lib/components/main-menu/main-menu.component.html

```diff
@@ -1,4 +1,4 @@
-<!-- Old Version Menu -->
+<!-- Without Dropdown Version -->
```

Comment:
Das ist jetzt eine gute Lösung.
Vielleicht können wir irgendwann ein _flag_ in der skeleton-config (?) einführen, um dieses Feature zu aktivieren. 
(aber nicht jetzt! Der PR muss raus 😉 )